libqtile/backend/wayland/inputs.py

```diff
@@ -142,6 +143,8 @@ class InputConfig(configurable.Configurable):
         ("kb_variant", None, "Keyboard variant i.e. ``XKB_DEFAULT_VARIANT``"),
         ("kb_repeat_rate", 25, "Keyboard key repeats made per second"),
         ("kb_repeat_delay", 600, "Keyboard delay in milliseconds before repeating"),
+        ("kb_numlock", None, "``True`` or ``False`` Enable numlock on startup"),
+        ("kb_capslock", None, "``True`` or ``False`` Enable capslock on startup"),
```

Comment:
Can you make the default `False` here instead of `None`?

Reply:
sure.

Reply:
I would argue that num lock should be true by default

Reply:
In that case wouldn't it be better to set the numlock to turn on in the default config instead?

Reply:
that is a good idea